metabase/metabase
# MS3: Open notebook data source in new tab (tooltip)

### Description
This PR addresses the milestone 3 from #46201.
It implements the tooltip that instructs users to click (with a modifier) on a data source in order to open it in a new tab/window. The contents of the tooltip depends on the operating system. It shows `⌘` for mac os, and `Ctrl` for Linux and Windows.

![image](https://github.com/user-attachments/assets/0e8b5dc4-2dfd-4259-bf17-f1b5fe2c4293)

![image](https://github.com/user-attachments/assets/ee775090-cf72-4390-b6e9-847c88d2009c)

### How to verify
1. Open any entity (except Metrics) in the notebook mode (raw table, question, model)
2. Hover the data source
3. A tooltip should appear
4. Add a join (any type of join)
5. The moment you select a right table for the join, you can hover it and the same tooltip should appear

Alternatively, run the accompanied tests.

### Demo
https://github.com/user-attachments/assets/d9c30c1d-33e7-4ebc-9707-2e29cf0fad2f

### Checklist
